api/HTMLOListElement.json

```diff
@@ -124,10 +124,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "5"
+              "version_added": "6"
```

Comment:
Where did the original "5" come from?

Reply:
Another entry from the wiki migration, looks like.

Reply:
Thanks for checking!